test/aliases.js

```diff
@@ -0,0 +1,59 @@
+import { join } from 'path'
+import { version } from 'process'
+
+import test from 'ava'
+
+import { resolveVersionRangeAlias } from '../src/aliases.js'
+
+const resolveInFolder = (versionRange, folder) =>
+  resolveVersionRangeAlias(versionRange, {
+    cwd: join(__dirname, 'fixtures', folder),
+  })
+
+test('Resolve - with node-version pseudo version', async (t) => {
+  const versionRange = await resolveInFolder('.', 'node-version-project')
+  t.is(versionRange, 'v12.12', 'nvmrc version not resolved')
```

Comment:
I think you should be able to drop the error message in this file, since they mostly repeat the test title.

Reply:
(Note: by "error message", I meant the third argument to `t.is()`.)